docs/topics/lombok.md

```diff
@@ -0,0 +1,149 @@
+[//]: # (title: Lombok compiler plugin)
+
+> The Lombok compiler plugin is [Experimental](components-stability.md). 
+> It may be dropped or changed at any time. Use it only for evaluation purposes. 
+> We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-7112).
+>
+{type="warning"}
+
+Since 1.5.20, Kotlin supports the experimental Lombok Kotlin compiler plugin. This plugin allows the use of
+Java's Lombok-generated declarations in mixed Kotlin/Java sources.
```

Comment:
I'll repeat my concern from above: let's be as specific as possible. If it's about using Lombok-generated declarations from **Kotlin sources within the same module**, we should say exactly this.

Reply:
Does it work?

>The Lombok compiler plugin allows the generation and use of Java's Lombok declarations within the same multilingual module. 
If such declarations are generated in another module than the Kotlin code that uses these declarations, 
then you don't need to use this plugin.

Reply:
Also, I've put a suggestion to fix the same in What's new Google doc.

Reply:
See my answer above about "multilingual module". Otherwise good.